games/mariobros/js/mariobros.json

```diff
@@ -6,9 +6,9 @@
 		{"filename":"btn_gameb","frame":{"x":151,"y":131,"w":41,"h":60},"rotated":false,"trimmed":false,"spriteSourceSize":{"x":816,"y":450,"w":41,"h":60},"sourceSize":{"w":41,"h":60}},
 		{"filename":"btn_time","frame":{"x":109,"y":131,"w":41,"h":60},"rotated":false,"trimmed":false,"spriteSourceSize":{"x":875,"y":450,"w":41,"h":60},"sourceSize":{"w":41,"h":60}},
 		{"filename":"btn_luigi_up","frame":{"x":175,"y":0,"w":58,"h":129},"rotated":false,"trimmed":false,"spriteSourceSize":{"x":174,"y":468,"w":58,"h":129},"sourceSize":{"w":58,"h":129}},
-		{"filename":"btn_luigi_dn","frame":{"x":116,"y":0,"w":58,"h":129},"rotated":false,"trimmed":false,"spriteSourceSize":{"x":174,"y":468,"w":58,"h":129},"sourceSize":{"w":58,"h":129}},
```

Comment:
`btn_luigi_dn` and `btn_mario_dn` were the only frames not using the `_down` suffix, renamed for consistency.